spidermon/contrib/scrapy/pipelines.py

```diff
@@ -84,7 +85,9 @@ def set_validators(loader, schema):
     @classmethod
     def _load_jsonschema_validator(cls, schema):
         if isinstance(schema, six.string_types):
-            if schema.endswith(".json"):
+            if web.is_url(schema):
+                schema = json.loads(web.get_contents(schema))
```

Comment:
What if the content returned is not a valid JSON?

Reply:
You will get an exception. How it's supposed to work?
Probably I need to put .json back, at least it's dummy proof.

Reply:
You can be friendly to the user with something like:
```python
try:
     schema = json.loads(web.get_contents(schema))
except json.JSONDecodeError:
     self.logger.exception(f'Could not parse json schema downloaded from {schema}')
```

Reply:
I agree with @raphapassini . In this case it would be better to provide a better error message to the user being more explicit over the problem that occurred.

Reply:
How do I access logger?